wpilibc/athena/src/RobotDrive.cpp

```diff
@@ -625,14 +610,15 @@ void RobotDrive::SetLeftRightMotorOutputs(double leftOutput,
 /**
  * Limit motor values to the -1.0 to +1.0 range.
  */
-double RobotDrive::Limit(double num) {
-  if (num > 1.0) {
-    return 1.0;
-  }
-  if (num < -1.0) {
-    return -1.0;
-  }
-  return num;
+double RobotDrive::Limit(double number) {
```

Comment:
Can't really argue with that.  When I see this pattern it jumps out as performing a constrain.